src/unlimited.cpp

```diff
@@ -220,7 +228,7 @@ std::string FormatCoinbaseMessage(const std::vector<std::string> &comments, cons
             ss << "/" << *it;
         ss << "/";
     }
-    std::string ret = ss.str() + minerComment;
+    std::string ret = ss.str() + customComment;
```

Comment:
isn't this a bug in the current code, since we are referring to `minerComment` global var rather than the `customComment` parameter?

Reply:
yes, its a bug in specification but not in practice because every call to FormatCoinbaseMessage passes "minerComment" as the "customComment".